Eastern-Research-Group/mywaterway
# Feature/281 physical chemical profile

## Related Issues:
* [HMW-281](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-281)

## Main Changes:
* Switched from the Narrow WQX data profile to the Physical/Chemical data profile.
* Added a select input for the "ActivityMediaName" field of the Physical/Chemical profile.
* Added a line to filter out records according to the value of their "ActivityTypeCode" field.
* Removed the select input for the "MethodSpecificationName" field values, which is absent in the Physical/Chemical profile.
* Refactored to flatten a nested data structure.

## Steps To Test:
1. Go to a _Monitoring Location Details_ page, e.g. [http://localhost:3000/location/STORET/CBP_WQX/CBP_WQX-ANA24/](http://localhost:3000/location/STORET/CBP_WQX/CBP_WQX-ANA24/)
2. Select a characteristic from the table with radio inputs (one with data that can be charted)
3. Observe that the "Speciation" drop-down no longer exists above the chart, but a "Media Name" drop-down does.
4. Ensure that the refactored functionality still works as expected (the characteristic chart data).
